cmd/kubeadm/app/componentconfigs/kubelet_test.go

```diff
@@ -106,6 +107,7 @@ var kubeletMarshalCases = []struct {
 			port: 12345
 			rotateCertificates: true
 			runtimeRequestTimeout: 0s
+			runtimeSandBoxTimeout: 0s
```

Comment:
hi, these changes in cmd/kubeadm are no longer required.

as you can see kubeletMarshalCases no longer exists here:
https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/componentconfigs/kubelet_test.go


/remove-area kubeadm
/remove-sig cluster-lifecycle